examples/next/getting-started/tsconfig.json

```diff
@@ -18,7 +18,8 @@
     "isolatedModules": true,
     "jsx": "preserve",
     "baseUrl": "./src",
-    "incremental": true
+    "incremental": true,
+	"typeRoots": ["node_modules/@types"]
```

Comment:
Prevents type conflicts between different React Versions. So we need to specify explicitly the location of `typeRoots`.